db/migrate/20210615101111_update_ons_incedence_dataset_cas_type.rb

```diff
@@ -0,0 +1,10 @@
+# plan.io 25866 - set ONS incedence dataset cas_type to nil
+class UpdateOnsIncedenceDatasetCasType < ActiveRecord::Migration[6.0]
+  include MigrationHelper
+  def change
+    id = Dataset.find_by(name: 'ONS incidence tables (all available)Schema: ONS1971_1994, 1989, ' \
```

Comment:
we ok on live db, but we need to guard against possible duplicate names in production dbs in the future